pennylane_qiskit/qiskit_device.py

```diff
@@ -12,525 +12,664 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 r"""
-This module contains a base class for constructing Qiskit devices for PennyLane.
+This module contains a prototype base class for constructing Qiskit devices
+for PennyLane with the new device API.
 """
-# pylint: disable=too-many-instance-attributes,attribute-defined-outside-init
+# pylint: disable=too-many-instance-attributes,attribute-defined-outside-init, missing-function-docstring
```

Comment:
Why do we need `missing-function-docstring`? Can we not assign docstrings to all the functions here?

Reply:
I think you can safely delete this without issue since `execute()` has a docstring now.